views/layout.jade

```diff
@@ -19,6 +19,9 @@ html
     link(rel='stylesheet', href='/assets/bootstrap/css/bootstrap.min.css')
     link(rel='stylesheet', href='/content/wicked.css')
     link(rel="stylesheet", href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.3.0/styles/default.min.css")
```

Comment:
While you're at it, would you mind also replacing highlight.js with its npm counterpart and also serving it via the `/assets` end point?